import/dproto/attributes.d

```diff
@@ -40,6 +42,8 @@ template TagId(alias T)
 template ProtoAccessors()
 {
 
+@safe:
+
```

Comment:
This shouldn't be applied to the templated methods; you can't guarantee it is appropriate.

I would suggest it only be applied to the non-templated `serialize()` method.  Attribute inference will take care of the rest.